api/views/andela_centres.py

```diff
@@ -36,12 +38,48 @@ class AndelaCentreViewset(ModelViewSet):
     permission_classes = [IsAuthenticated, IsAdminUser]
     authentication_classes = [FirebaseTokenAuthentication]
 
+    def create(self, request, *args, **kwargs):
+        if request.user.is_superuser:
+            return super().create(request, *args, **kwargs)
+        else:
+            raise PermissionDenied("Only a super user can create a center")
+
     def destroy(self, request, *args, **kwargs):
         instance = self.get_object()
         self.perform_destroy(instance)
         data = {"detail": "Deleted Successfully"}
         return Response(data=data, status=status.HTTP_204_NO_CONTENT)
 
+    @action(
+        detail=True,
+        permission_classes=[IsAuthenticated, IsAdminUser],
+        serializer_class=OfficeBlockSerializer,
+    )
+    def office_blocks(self, request, pk=None):
```

Comment:
I think this can be handled in the office block serializer.

Reply:
@egichuri .. we are getting office blocks in a center. I put it under the center serializer because the URL is supposed to be in the format of `centre/<center-id>/office_block`..blocks belong to a center, so this has to be the right format.and also the center generates the URL for us, the 
 center serializer handles filtering and validating if a center exists or not.

Reply:
ok.. sounds good